### PR TITLE
feat: structure-links

### PR DESCRIPTION
**Pull request by matuz**

### CODE REVIEW

File: src/erudio-client.ts [hunk 1]
```diff
@@ -25,6 +27,31 @@ export class ErudioClient {
     return new Structure(this.host).listNodes(namespace, options);
   };
 
+  public getStructureNodes = async (
```
Comment (matuz) [inline src/erudio-client.ts]: this method is going to be renamed to `getStructureLinks`

File: src/structure-link/structure-link.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  public async getOne(id: string): Promise<StructureLinkDto> {
+    const response = await new HttpClientProxy().get<StructureLinkDto[]>(
+      this.url,
+      {
+        params: { id },
+      },
+    );
+
+    if (!response.length) {
+      throw {
```
Comment (JERILJOSE96) [inline src/structure-link/structure-link.ts:31]: Use HttpClientProxyError for consistency

Comment (matuz) [reply src/structure-link/structure-link.ts:31]: This casting is the best I can do, as `HttpClientProxyError` is only as interface.

File: README.md [hunk 3]
```diff
@@ -6,7 +6,22 @@ Erudio HTTP library to fetch data from various endpoint/GraphQL
...
+
+// Get single Node filled with data, for given structureId
 const singleNode = erudioClient.getStructureNode( 'f07e796f-e3bd-46f0-97a5-c3ae7409dc', 'en');
-const tags = erudioClient.getTags(ServiceType.CONTENT, 'f07e796f-e3bd-46f0-97a5-c3ae7409da')
+
+// Get all the tags for given object type and object id.
+const tags = erudioClient.getTags(ServiceType.CONTENT, 'f07e796f-e3bd-46f0-97a5-c3ae7409da');
+
+// Get all the links by given namespaceId
+const links = erudioClient.getStructureLinks({targetNamespaceId: 'f07e796f-e3bd-46f0-97a5-c3ae7409dc'});
```
Comment (espresse) [inline README.md:20]: @matuz @iapain we should have some consistency in the API: `getStructures` have `namespaceId` as its first parameter, then `options`, here we have an object with `targetNamespaceId` key (which IMO should be rather `namespace` or `namespaceId` if we decide to go with passing an object here)

Comment (espresse) [reply README.md:20]: ok, now I see it's becuase of the way params are defined in the structure link service

**Review by espresse — APPROVED** [review]

**Review by iapain — APPROVED** [review]
@matuz Thanks!

**Pull request merged**